test/videojs-contrib-hls.test.js

```diff
@@ -952,7 +952,7 @@ QUnit.test('fire loadedmetadata once we successfully load a playlist', function(
   });
   openMediaSource(this.player, this.clock);
   this.player.tech_.hls.bandwidth = 20000;
-  this.player.on('loadedmetadata', function() {
+  this.player.tech_.hls.masterPlaylistController_.on('loadedmetadata', function() {
```

Comment:
Hmm, this test might need to be rethought completely.. It really isn't testing anything now but I'm not sure if it is possible to test that `loadedmetadata` is being fired by the video element without appending some real data.